wayland-scanner/src/main/java/org/freedesktop/wayland/generator/impl/ProtocolXmlPathResolver.java

```diff
@@ -1,29 +1,50 @@
 package org.freedesktop.wayland.generator.impl;
 
+import java.io.BufferedReader;
```

Comment:
Please do not include this diff noise caused by differences in the formatting configuration :)